test/e2e/__snapshots__/history-api-fallback.test.js.snap.webpack5

```diff
@@ -65,7 +65,11 @@ exports[`historyApiFallback option as object with static and rewrites historyApi
 "
 `;
 
-exports[`historyApiFallback option as object with static set to false historyApiFallback should work and ignore static content: console messages 1`] = `[]`;
+exports[`historyApiFallback option as object with static set to false historyApiFallback should work and ignore static content: console messages 1`] = `
+[
+  "Failed to load resource: the server responded with a status of 404 (Not Found)",
```

Comment:
- The [new headless mode](https://developer.chrome.com/articles/new-headless/) captures this when favicon is not present. It's valid here as static is false.